src/main/java/com/example/java/maven/gameBlackJack/Croupier.java

```diff
@@ -0,0 +1,28 @@
+package com.example.java.maven.gameBlackJack;
+
+import java.util.ArrayList;
+
+public class Croupier {
+    String name;
+    ArrayList<Card> hand;
+    int points;
```

Comment:
do you really need this field? You always starting from 0, when you are using `getPoints()` method

Reply:
Done